### PR TITLE
feat: notification nav footer item and page

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- Base UI for the notifications page
- Nav bar change to include the notifs items.

Note: the number indication is not yet available until such integration exist.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-696 #done

### CODE REVIEW

File: packages/webapp/pages/history.tsx [hunk 1]
```diff
@@ -50,24 +51,12 @@ const History = (): ReactElement => {
...
-        {seo}
-        <ReadingHistoryEmptyScreen />
-      </>
-    );
-  }
-
   return (
-    <>
-      {seo}
+    <ProtectedPage
```
Comment (rebelchris) [inline packages/webapp/pages/history.tsx:55]: Just wondering if the layouts won't be a better option for this?

**Review by rebelchris — APPROVED** [review]
Nice work on this 🥳

**Comment by rebelchris** [comment]
@sshanzel I'm not sure you want this to go into main though?
Should we not opt for a intermediate branch, else this gets added in the frontend for users.

**Comment by sshanzel** [comment]
> @sshanzel I'm not sure you want this to go into main though?
Should we not opt for a intermediate branch, else this gets added in the frontend for users.

Ahh, my bad. Yep I created an intermediate branch earlier but I forgot to point this one to that.

File: packages/webapp/pages/history.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/webapp/pages/history.tsx:55]: Are you referring to the function `getLayout`? Or the NextJS layout?

Comment (rebelchris) [reply packages/webapp/pages/history.tsx:55]: Basically the Next layouts indeed, we could opt to create a ProcetedPage layout that could wrap around everything?
But open to this one as well so leaving it up to you.

Comment (sshanzel) [reply packages/webapp/pages/history.tsx:55]: Ah, interesting. Will look into that, might be something quick we can implement, at least we can start now.

**Review by el — APPROVED** [review]

File: packages/webapp/pages/history.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/webapp/pages/history.tsx:55]: We are already utilizing the `getLayout` property, the current implementation should be fine with its own layout as the `getLayout` will provide the nesting layout.

**Pull request merged**